Doc/whatsnew/3.10.rst

```diff
@@ -237,6 +242,11 @@ New Features
   :class:`datetime.time` objects.
   (Contributed by Zackery Spytz in :issue:`30155`.)
 
+* Add a :c:func:`PyCodec_Unregister` to unregister a codec search function.
+  (Contributed by Hai Shi in :issue:`41842`.)
+
+=======
```

Comment:
What is the meaning of this line? If it's a mistake, please remove it.

Reply:
updated.